src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -750,49 +781,55 @@ private void modifyGeneratorMinMaxReactiveLimits(GeneratorModificationInfos modi
             }
         }
 
-        // (if the min and max reactive limits are null and there is no previous min max limits set we set them to Double max and
+        // (if the min and max reactive limits are null and there is no previous min max
+        // limits set we set them to Double max and
         // Double min values)
```

Comment:
While you were re-wrapping, you could lift up "Double min values" to the line above

Reply:
Done